deploy/cloudwash_job.yaml

```diff
@@ -0,0 +1,72 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: provisioning-cleanup-resources
+objects:
+- apiVersion: batch/v1
+  kind: Job
+  metadata:
+    name: provisioning-cleanup-${IMAGE_TAG}-${UID}
+    annotations:
+      "ignore-check.kube-linter.io/no-liveness-probe": "probes not required on Job pods"
+      "ignore-check.kube-linter.io/no-readiness-probe": "probes not required on Job pods"
+  spec:
+    backoffLimit: 0
+    template:
+      spec:
+        imagePullSecrets:
+        - name: quay-cloudservices-pull
+        restartPolicy: Never
+        containers:
+        - name: provisioning-cleanup-${IMAGE_TAG}-${UID}
+          image: quay.io/redhatqe/cloudwash
+          imagePullPolicy: Always
+          args:
+          - swach -d azure --all # remove "-d (dry run)" after overall review.
+#          - swach -d aws --all
+#          - swach -d gce --all
+          env:
+          - name:  CLEANUP_PROVIDERS_AZURE_AUTH_CLIENT_ID
+            valueFrom:
+              secretKeyRef:
+                key: azure_client_id
+                name: iqe-vault
+                optional: true
+          - name: CLEANUP_PROVIDERS_AZURE_AUTH_SECRET_ID
+            valueFrom:
+              secretKeyRef:
+                key: azure_client_secret
+                name: iqe-vault
+                optional: true
+          - name: CLEANUP_PROVIDERS_AZURE_AUTH_TENANT_ID
+            valueFrom:
+              secretKeyRef:
+                key: azure_tenant_id
+                name: iqe-vault
+                optional: true
+          - name: CLEANUP_PROVIDERS_AZURE_AUTH_SUBSCRIPTION_ID
+            valueFrom:
+              secretKeyRef:
+                key: subscription_id
+                name: iqe-vault
```

Comment:
Do we have secret in vault called iqe-vault containing all the values in json? :open_mouth:

Reply:
Yep here https://vault.devshift.net/ui/vault/secrets/insights/list/secrets/qe/qa/provisioning/

Reply:
it's called azure there :thinking: we rename it to iqe-vault in app-interface?
I'd say more descriptive name would be slightly better :stuck_out_tongue:

Reply:
Yeah I think you are right 
I can confirm this.

I was mostly studying other files and writing our own so they have the same thing `iqe vault` everywhere. 
then I thought it might be some sort of keyword to connect vault.
 
 for example here 
 https://github.com/RHEnVision/provisioning-backend/blob/main/deploy/testing-integration.yaml#L54